### PR TITLE
kubeadm: update OWNERS for 1.16

### PR DESCRIPTION
**Pull request by neolit123**
**What this PR does / why we need it**:
- comment out Liz and Chuck until further notice.
Feel free to come back to kubeadm!!
- Add SataQiu as reviewer. Welcome.
- Add ereslibre as approver. Congrats

SataQiu's PR to self promote.
https://github.com/kubernetes/kubernetes/pull/79167

**Which issue(s) this PR fixes**:

NONE

**Special notes for your reviewer**:
NONE

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

@kubernetes/sig-cluster-lifecycle-pr-reviews 
/hold
/assign @luxas @timothysc @fabriziopandini 
/kind cleanup
/priority important-soon

### CODE REVIEW

**Comment by neolit123** [comment]
cc @liztio @chuckha

**Comment by SataQiu** [comment]
Thank you @neolit123
/LGTM

**Comment by chuckha** [comment]
👍

**Comment by detiber** [comment]
:+1: from me

**Comment by fabriziopandini** [comment]
+1 for me

**Review by rosti — APPROVED** [review]
+1 from me!

Thank you @liztio and @chuckha for your time on kubeadm!
Welcome @SataQiu and thank you for volunteering!

File: cmd/kubeadm/OWNERS [hunk 1]
```diff
@@ -12,14 +12,13 @@ reviewers:
...
 - kad
-- liztio
-- chuckha
 - detiber
 - dixudx
 - rosti
 - yagonobre
-- ereslibre 
-# Might want to add @xiangpengzhao and @stealthybox back in the future
+- ereslibre
```
Comment (timothysc) [inline cmd/kubeadm/OWNERS]: We should also consider grooming @ereslibre for approver imo.

Comment (neolit123) [reply cmd/kubeadm/OWNERS]: +1 from me. should be voted in a meeting? can be added in a separate PR?

also most importantly we should ask @ereslibre if he's ok with that.
at the EOD, reviewers + approvers need to have the bandwidth.

Comment (ereslibre) [reply cmd/kubeadm/OWNERS]: It's true that I've been a bit away for some time, but planning to allot some time daily to review PR's, I'd agree if folks want.

Comment (timothysc) [reply cmd/kubeadm/OWNERS]: Let's final vote this week in the meeting then unhold this PR.

**Review by luxas — APPROVED** [review]
Thanks for this @neolit123!
Let me know when you've resolved the comments in the discussion below.

Please remember to reflect/sync these changes in/between https://github.com/kubernetes/kubeadm/blob/master/OWNERS

**Comment by liztio** [comment]
/lgtm 

my most bittersweet lgtm :crying_cat_face:

**Comment by neolit123** [comment]
updated to:
- comment out Liz and Chuck until further notice.
Feel free to come back to kubeadm!!
- Add SataQiu as reviewer. Welcome.
- Add ereslibre as approver. Congrats

me and @timothysc are +1 on this as discussed during today's kubeadm office hours.

merging on lazy consensus next week. please state objections, if any.

**Comment by yagonobre** [comment]
+1 from me

**Review by fabriziopandini — APPROVED** [review]
Many many thanks @liztio and @chuckha! 
Congrats to @SataQiu and @ereslibre!
/lgtm

**Pull request merged**